### DISCUSSION TITLE
NuGet manager doesn't detect all 'Directory.Build.props' files

### DISCUSSION THREAD

**Discussion by felpel** [renovatebot/renovate#17390] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

32.174.1

### Please select which platform you are using if self-hosting.

Azure DevOps (dev.azure.com)

### If you're self-hosting Renovate, tell us what version of the platform you run.

Azure DevOps

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

Given a repository with multiple `Directory.Build.props` files, Renovate doesn't seem to detect one of them for the `nuget` manager.

The folder structure looks like this:

```
src/
├─ backend/
│  ├─ src/
│  ├─ tests/
│  │  ├─ Directory.Build.props
│  ├─ Directory.Build.props
```

The one under `src/backend` is detected, but the other isn't (see logs).

I would normally expect that all `Directory.Build.props` files are detected, not only the first occurrence.

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: Matched 20 file(s) for manager nuget: src/backend/Directory.Build.props, {all csproj redacted}, (repository={repository name redacted})
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment, marked as answer]
tests directories are ignored by default 

https://docs.renovatebot.com/presets-default/#ignoremodulesandtests

**Reply by felpel** [reply]
@viceice Do you have any tip on how to configure Renovate if the `config:base` preset is present in `extends`, but allow the ` "**/tests/**"` folder(s)?

**Reply by rarkins** [reply]
Redefine `ignorePaths` to override the values in `config:base`

**Reply by viceice** [reply]
https://docs.renovatebot.com/configuration-options/#ignorepresets

**Reply by felpel** [reply]
Let me try that out and confirm what was the final solution. Thank you for your help!

**Reply by felpel** [reply]
@viceice @rarkins sorry about the late response, but my issue was definitely caused by my misunderstanding of presets and how I could override the values. We're good now 👍
